WcaOnRails/config/locales/en.yml

```diff
@@ -708,10 +708,15 @@ en:
         mail_subject: "You have registered for %{comp_name}"
         #context: "comp_name" contains the competition's name
         registered_html: "You just registered for %{comp_name}."
-        #context: "n_total" will be replaced by the total number of registrations on waiting list, and "people" by the pluralization of "person" in your language (key registrations.registration_info_people.person)
-        on_waiting_list: "Your registration is on the waiting list, which currently has %{n_total} %{people} on it."
+        awaits_approval: "Your registration has been accepted by the system and is pending approval from the organisation team."
```

Comment:
nit: As part of incorporating in the US, we decided to use US English throughout the website, so this should actually be "organization team", sorry @Mollerz =(